### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by NTanatar**

### CODE REVIEW

File: src/test/java/com/tannat/country/services/impl/CityServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    @Mock
+    private CityRepository cityRepository;
+
+    @InjectMocks
+    private CityServiceImpl cityService;
+
+    private final City cityA = City.builder().id(1L).name("A").population(456).build();
+    private final City cityB = City.builder().id(2L).name("B").population(11).build();
+    private final City createdCity = City.builder().id(22L).name("CreatedCity").build();
+    private final City updatedCity = City.builder().id(23L).name("UpdatedCity").build();
```
Comment (Andriireva) [inline src/test/java/com/tannat/country/services/impl/CityServiceImplTest.java]: lets make final field capitalized

File: src/test/java/com/tannat/country/services/impl/CountryServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+            City.builder().id(3L).name("Hill").countryId(countryF.getId()).build(),
+            City.builder().id(5L).name("Lake").countryId(countryF.getId()).build());
+
+    private final List<City> citiesO = Collections.singletonList(
+            City.builder().id(11L).name("Atlantida").countryId(countryO.getId()).build());
+
+    private final Country createdCountry = Country.builder().id(102L).name("Created").build();
+    private final Country updatedCountry = Country.builder().id(103L).name("Updated").build();
+
+    private final City createdCity = City.builder().id(22L).countryId(102L).name("CreatedCity").build();
```
Comment (Andriireva) [inline src/test/java/com/tannat/country/services/impl/CountryServiceImplTest.java]: lets make fields capitalized

**Review by Andriireva — CHANGES_REQUESTED** [review]

File: src/test/java/com/tannat/country/services/impl/CityServiceImplTest.java (hunk 1 above)
Comment (NTanatar) [reply src/test/java/com/tannat/country/services/impl/CityServiceImplTest.java]: done

File: src/test/java/com/tannat/country/services/impl/CountryServiceImplTest.java (hunk 2 above)
Comment (NTanatar) [reply src/test/java/com/tannat/country/services/impl/CountryServiceImplTest.java]: done

**Review by Andriireva — APPROVED** [review]

**Pull request merged**